### ISSUE TITLE
vpc_dns.yml schema doesn't have domain/hostname indicator

### ISSUE THREAD

**Issue by slw07g** [panther-labs/panther-analysis#382]
https://github.com/panther-labs/panther-analysis/blob/6bfdd1357668bd015628471302a4a29a42a03e1b/schemas/logs/aws/vpc_dns.yml#L34-L37

Seems like this is a good place to have a domain/hostname indicator.

**Comment by rleighton** [comment]
That is true and we have one implicitly. The parser has special code to process the query name as well as extract indicators in the answers.
